multi-agent/adapters/inbound/flask/endpoints/sessions.py

```
@@ -248,10 +249,30 @@ def get_session_status(session_id):
+        if paginated:
+            total = svc.count(identity, filters)
+            return jsonify({
+                "sessions": items,
+                "pagination": {
+                    "total": total,
+                    "limit": limit,
+                    "offset": offset,
+                    "has_more": offset + limit < total,
+                },
+            }), 200
+        else:
```

Ruff (0.16.1)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)